### ISSUE TITLE
Come up with a name

### ISSUE THREAD

**Issue by aj-r** [red3141/UrfChallenge2015#14]
Come up with a name for the game. Probably get a domain based on this name (e.g. from freenom).

It would be nice to have the domain before we start user testing, but not totally required.

**Comment by aj-r** [comment]
We have decided on "Phantasmagoria of Silver Spatula"
